etc/kayobe/kolla/globals.yml

```diff
@@ -4,42 +4,63 @@
 # To work around issue of trying to install docker from
 # empty pulp server, use upstream docker dnf repo on
 # non-overcloud hosts
-enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names or ansible_facts.os_family == 'Debian' }}"{% endraw %}
-
-
-{% if kolla_base_distro == 'centos' %}
```

Comment:
A recent PR is causing a merge conflict: https://github.com/stackhpc/stackhpc-kayobe-config/pull/516

Need to update the tags to match.

Reply:
Done

Reply:
Not seeing this yet

Reply:
My mistake, I'll rebase and try again

Reply:
should be good now